litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/styles.ts

```diff
@@ -61,6 +61,7 @@ const useStyles = makeStyles((theme) => ({
   // Modal
   closeButton: {
     borderColor: theme.palette.border.main,
+    marginTop: theme.spacing(-3.75),
```

Comment:
create an issue also in litmus-ui for this negative margin

Reply:
Sure. I will try to remove this negative margin with some fix. Thanks @ishangupta-ds

Reply:
No it's fine here, no need to deviate from the design but please create a issue or add link to this PR to exiting modal issue on litmus-ui